superflore/generators/bitbake/yocto_recipe.py

```diff
@@ -499,24 +500,42 @@ def get_recipe_text(self, distributor):
         ret += '\n' + self.get_bottom_inherit_line()
         return ret
 
-    @staticmethod
-    def _get_cache_last_modified_time(distro):
-        try:
-            from urllib.request import urlopen
-        except ImportError:
-            from urllib2 import urlopen
-        dist = get_index(get_index_url()).distributions[distro]
-        conn = urlopen(dist['distribution_cache'])
-        last_modified = conn.headers['last-modified']
-        ts = strptime(last_modified, '%a, %d %b %Y %H:%M:%S %Z')
-        return strftime('%Y%m%d%H%M%S', ts)
-
     @staticmethod
     def _get_ros_version(distro):
         distros = get_distros()
         return 2 if distro not in distros \
             else int(distros[distro]['distribution_type'][len('ros'):])
 
+    @staticmethod
+    def generate_superflore_datetime_inc(basepath, dist):
+        datetime_dir = '{}/conf/ros-distro/include/{}/'.format(basepath, dist)
+        datetime_file_name = 'generated-superflore-datetime.inc'
+        datetime_path = '{}{}'.format(datetime_dir, datetime_file_name)
+        try:
+            make_dir(datetime_dir)
+            with open(datetime_path, 'w') as datetime_file:
+                datetime_file.write('# {}/{}\n'.format(
+                    dist, datetime_file_name))
+                datetime_file.write('# Generated by superflore -- DO NOT EDIT')
+                datetime_file.write(
+                    ' (except ROS_DISTRO_METADATA_VERSION_REVISION)\n#\n')
+                datetime_file.write(
+                    '# Copyright 2019 Open Source Robotics Foundation\n\n')
```

Comment:
Can we make this use the current year not hard code 2019 for future maintenance avoidance.

Reply:
@andre-rosa Please implement this and I'll merge it in

Reply:
Thanks @tfoote, makes sense.